MantidQt/CustomInterfaces/src/Muon/MuonAnalysis.cpp

```diff
@@ -1528,7 +1528,14 @@ void MuonAnalysis::updateFrontAndCombo(bool updateIndexAndPlot) {
   if (currentI >= m_uiForm.frontGroupGroupPairComboBox->count()) {
     currentI = 0;
   }
-  m_uiForm.fitBrowser->setAvailableGroups(groupsAndPairs);
+  auto tmp = m_groupingHelper.parseGroupingTable().groupNames;
+  auto names = m_groupingHelper.parseGroupingTable().pairNames;
+  names.insert(std::end(names), std::begin(tmp), std::end(tmp));
```

Comment:
changing it to 

```
auto names = m_groupingHelper.parseGroupingTable().pairNames;
	names.insert(std::end(names), std::make_move_iterator(m_groupingHelper.parseGroupingTable().groupNames.begin()), std::make_move_iterator(m_groupingHelper.parseGroupingTable().groupNames.end()));
	QStringList GroupsAndPairsNames;
```

causes an error. I think we do want a copy, as the group names should still contain the names.